### ISSUE TITLE
Predictive Maintenance Equipment Instruments

### ISSUE THREAD

**Issue by ldneal** [abhisheks008/ML-Crate#633]
### ML-Crate Repository (Proposing new issue)
:red_circle: **Project Title** : Predictive Maintenance Equipment Instrument
:red_circle: **Aim** : To check the maintenance of Equipment in Instrument Power Plant
:red_circle: **Dataset** :
:red_circle: **Approach** : Try to use 3-4 algorithms to implement the models and compare all the algorithms to find out the best fitted algorithm for the model by checking the accuracy scores. Also do not forget to do a exploratory data analysis before creating any model.

**********************************************************************************
### 📍 **Follow the Guidelines to Contribute in the Project :**
* You need to create a separate folder named as the Project Title.
* Inside that folder, there will be four main components.
   * Images - To store the required images.
   * Dataset - To store the dataset or, information/source about the dataset.
   * Model - To store the machine learning model you've created using the dataset.
   * `requirements.txt` - This file will contain the required packages/libraries to run the project in other machines.
* Inside the `Model` folder, the `README.md` file must be filled up properly, with proper visualizations and conclusions.

***********************************************************************
:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
- "Issue Title" and "PR Title should be the same. Include issue number along with it.
- Follow Contributing Guidelines & Code of Conduct before start Contributing.

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : Neal Gandhi
- GitHub Profile Link : https://github.com/ldneal
- Participant ID (If not, then put NA) : Contributor
- Approach for this Project : Own SSOC
- What is your participant role? (Mention the Open Source Program name. Eg. HRSoC, GSSoC, GSOC etc.)

*************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by ldneal** [comment]
Sir I want to be work in this project would you assign me

**Comment by abhisheks008** [comment]
You are already assigned to an issue. Please complete that one first.

**Comment by ldneal** [comment]
Ok Sir

**Comment by siddhant4ds** [comment]
@abhisheks008 If this issue is still available, I would like to work on it.

Since there is no dataset associated with this issue, I suggest this one: [Predictive Maintenance Dataset (AI4I 2020)](https://www.kaggle.com/datasets/stephanmatzka/predictive-maintenance-dataset-ai4i-2020)

Full name: Siddhant Tiwari
GitHub profile link: https://github.com/siddhant4ds
Participant ID: sid4ds (Devfolio), sid4ds (Discord)
Participant role: SSOC-3 Contributor
Approach:
1. EDA and preprocessing of the dataset to make it usable for different ML models
2. Models implemented - linear models, SVM, RandomForest, Gradient Boosting trees (XGBoost, LightGBM), Neural networks (MLP)
3. After all models are implemented, try ensembling methods like weighted-averaging and hill-climbing to improve performance.
4. Will try SOTA/experimental models like TabPFN.

**Comment by abhisheks008** [comment]
Assigned @siddhant4ds
